scripts/ci-install.sh

```diff
@@ -6,20 +6,20 @@ NVIM_TAG=${NVIM_TAG-nightly}
 
 os=$(uname -s)
 if [[ $os == Linux ]]; then
-  wget https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-linux64.tar.gz
+  wget "https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-linux64.tar.gz"
   tar -zxf nvim-linux64.tar.gz
   sudo ln -s "$PWD"/nvim-linux64/bin/nvim /usr/local/bin
   rm -rf "$PWD"/nvim-linux64/lib/nvim/parser
   mkdir -p ~/.local/share/nvim/site/pack/nvim-treesitter/start
   ln -s "$PWD" ~/.local/share/nvim/site/pack/nvim-treesitter/start
 elif [[ $os == Darwin ]]; then
-  curl -L https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-macos-arm64.tar.gz | tar -xz
+  curl -L "https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-macos-$(uname -m).tar.gz" | tar -xz
   sudo ln -s "$PWD"/nvim-macos/bin/nvim /usr/local/bin
```

Comment:
also needs to be `nvim-macos-arm64` (sorry, tried to fix it myself, but conflicts...)

same for the line below